src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.constants;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import frc.utils.alerts.Alert;
+
+import java.io.IOException;
+
+public class VisionConstants {
+
+	public static final String FILTERED_DATA_LOGPATH_ADDITION = "FilteredData/";
+
+	public static final String NON_FILTERED_DATA_LOGPATH_ADDITION = "NonFilteredData/";
+
+	public static final String SOURCE_LOGPATH_ADDITION = "VisionSource/";
+
+
+	private static final String APRIL_TAG_FIELD_CONFIG_FILE_PATH = "april_tag_field_config.json";
+
+	public static final AprilTagFieldLayout APRIL_TAG_FIELD_LAYOUT = getAprilTagFieldLayout();
+
+	private static AprilTagFieldLayout getAprilTagFieldLayout() {
+		try {
+			return AprilTagFieldLayout.loadFromResource(APRIL_TAG_FIELD_CONFIG_FILE_PATH);
+		} catch (IOException e) {
```

Comment:
I'm not sure I agree with the comment, if we have a more specific exception we can catch, it can help us handle different issues more precisely

Reply:
[Actually Microsoft disallows catching general exceptions in their code design](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1031) (for c# though). It would be harder to detect a bug, further down the road, if we'll catch--for say--`NullPointerException`s too.